libs/blocks/global-navigation/utilities/menu/menu.css

```diff
@@ -197,4 +201,51 @@
   .feds-promo--dark a:hover {
     text-decoration: underline;
   }
+
+
+  .feds-links-strip-wrapper {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+    border-top: solid 1px #f3f3f3;
+  }
+
+  .feds-links-strip {
+    width: 100%;
+    padding: 20px 0;
+    max-width: var(--feds-maxWidth-nav);
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .feds-links-strip > *:not(:last-child)::after {
```

Comment:
We should ideally avoid `*` selectors, as they are non-performant. In this case, we know the children's structure, you could simply do `.feds-linksStrip-item:not(:last-child):after`

Reply:
Done